app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(4)
```

Comment:
- [ ] The User model should have a method that returns 3 most recent posts, not 4, kindly adjust your code so the method returns 3 last posts
`posts.order('created_at Desc').limit(3)`

Reply:
well resolved! ✔️